React/Lesson37/first-react-app/src/Header/Header.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './Header.css'; 
+
+const Header = () => {
+  return (
+    <header className="custom-header">
+      <div className="work_hours">
```

Comment:
use one convection of naming classes in your app

sometimes you use `_` sometimes `-`

Reply:
Thank you, I will consider it!